Author: Daniel Baumann <mail@daniel-baumann.ch>
Description:
 Disable dbus, signal, and ptrace in the apparmor profiles until Debian has a
 recent enough apparmor version (#746764), thanks to Intrigeri
 <intrigeri@debian.org> (Closes: #750107).

diff -Naurp lxc.orig/config/apparmor/abstractions/container-base lxc/config/apparmor/abstractions/container-base
--- lxc.orig/config/apparmor/abstractions/container-base
+++ lxc/config/apparmor/abstractions/container-base
@@ -5,9 +5,9 @@
 
   # The following 3 entries are only supported by recent apparmor versions.
   # Comment them if the apparmor parser doesn't recognize them.
-  dbus,
-  signal,
-  ptrace,
+  #dbus,
+  #signal,
+  #ptrace,
 
   # ignore DENIED message on / remount
   deny mount options=(ro, remount) -> /,
diff -Naurp lxc.orig/config/apparmor/abstractions/container-base.in lxc/config/apparmor/abstractions/container-base.in
--- lxc.orig/config/apparmor/abstractions/container-base.in
+++ lxc/config/apparmor/abstractions/container-base.in
@@ -5,9 +5,9 @@
 
   # The following 3 entries are only supported by recent apparmor versions.
   # Comment them if the apparmor parser doesn't recognize them.
-  dbus,
-  signal,
-  ptrace,
+  #dbus,
+  #signal,
+  #ptrace,
 
   # ignore DENIED message on / remount
   deny mount options=(ro, remount) -> /,
diff -Naurp lxc.orig/config/apparmor/abstractions/start-container lxc/config/apparmor/abstractions/start-container
--- lxc.orig/config/apparmor/abstractions/start-container
+++ lxc/config/apparmor/abstractions/start-container
@@ -4,9 +4,9 @@
 
   # The following 3 entries are only supported by recent apparmor versions.
   # Comment them if the apparmor parser doesn't recognize them.
-  dbus,
-  signal,
-  ptrace,
+  #dbus,
+  #signal,
+  #ptrace,
 
   # currently blocked by apparmor bug
   mount -> /usr/lib/*/lxc/{**,},
